metadata/deploy-tenant.sh

```
@@ -106,323 +108,388 @@ EOL
-            local table_schema
-            table_schema=$(grep -A 2 "^table:" "$table_file" | grep "schema:" | sed 's/.*schema:[[:space:]]*\([^ ]*\).*/\1/' | tr -d '\r' | tr -d '"' | tr -d "'")
-
-            if [ -z "$table_schema" ]; then
-                table_schema="public"
-            fi
 
-            if [ -z "$table_name" ]; then
-                echo "Warning: Could not determine table name from $table_file, skipping"
-                continue
-            fi
+<<<<<<< HEAD
```

Shellcheck (0.11.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)